docs/index.md

```diff
@@ -26,12 +39,39 @@ In future the provider will be extended to support more of Juju's capabilities.
 
 The provider has a dependency on Juju CLI configuration store.
 
-In order to operate, the provider expects configuration to be found in one of the following locations:
+## Authentication
 
-* `$XDG_DATA_HOME/juju`
-* `~/.local/share/juju`
+There are 3 ways to define credentials for authentication with the juju controller you wish to use. They are displayed in order in which the provider looks for credentials.
```

Comment:
The credentials are about the cloud, not the controller, right? (I know they can be used with a controller or a model, but in all cases they are about access to the cloud, right? I think it's important to say this clearly or else people confuse the credentials that access to the cloud with the credentials that allow a user to log in to a controller.)